indico/modules/events/registration/client/js/form/fields/TextAreaInput.jsx

```diff
@@ -12,45 +12,35 @@ import {Form} from 'semantic-ui-react';
 import {FinalInput, validators as v} from 'indico/react/forms';
 import {Translate} from 'indico/react/i18n';
 
+import '../../../styles/regform.module.scss';
+
 import {mapPropsToAttributes} from './util';
 
 const attributeMap = {
   numberOfRows: 'rows',
-  numberOfColumns: 'cols',
 };
 
-export default function TextAreaInput({htmlName, disabled, ...props}) {
+export default function TextAreaInput({htmlName, disabled, title, isRequired, ...props}) {
   const inputProps = mapPropsToAttributes(props, attributeMap, TextAreaInput.defaultProps);
-  return <textarea name={htmlName} {...inputProps} disabled={disabled} style={{resize: 'none'}} />;
+  return <textarea name={htmlName} {...inputProps} style={{resize: 'none'}} />;
 }
 
 TextAreaInput.propTypes = {
   htmlName: PropTypes.string.isRequired,
   disabled: PropTypes.bool,
   numberOfRows: PropTypes.number,
-  numberOfColumns: PropTypes.number,
+  title: PropTypes.string.isRequired,
+  isRequired: PropTypes.bool.isRequired,
 };
 
 TextAreaInput.defaultProps = {
   disabled: false,
   numberOfRows: 2,
-  numberOfColumns: 60,
 };
 
 export function TextAreaSettings() {
   return (
     <Form.Group widths="equal">
-      <FinalInput
-        name="numberOfColumns"
```

Comment:
If you remove an option here, also remove it from the server-side validation schemas (same for the TextInput `length`)